src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -89,12 +93,12 @@ public Uri SourceUri
                     var symbolApiKey = getSymbolApiKey(symbolSource);
 
                     await PushSymbols(packagePath, symbolSource, symbolApiKey,
-                        noServiceEndpoint, symbolPackageUpdateResource,
-                        requestTimeout, log, tokenSource.Token);
+                        noServiceEndpoint, skipDuplicate, symbolPackageUpdateResource,
+                        requestTimeout, log, tokenSource.Token, explicitSnupkgPush);
                 }
             }
         }
-
+        [Obsolete("Consolidating to one PackageUpdateResource.Push method which has all parameters defined.")]
```

Comment:
nit: not sure of our style guidelines, but I'd imagine we keep some whitespace between methods?

Reply:
Yes, should be whitespace here. Just missed it